### ISSUE TITLE
another idea

### ISSUE THREAD

**Issue by AfterNetwork** [Semisol/Smile.js#53]
This ideas involves scratch, which I am banned from so this would mostly require you.
The idea revolves around all the currenicies being dead and not much freedom of what is considered a shop.

(not main idea) new currency modal: The currency system is now refined and now actually uses it. If a shop joins a currency, the currency will require at least one item to be put in so currency is actually in circulation in there. The idea of inflation is also gone, as there is a max cap of overall currency and user's currency. Exclusives that use the currency cannot be given special treatment.

(main idea): A forum where people can freely make what they feel like is supposed to be in requests, and link everything together.

**Comment by AfterNetwork** [comment]
@Semisol

**Comment by Semisol** [comment]
> This ideas involves scratch, which I am banned from so this would mostly require you.
> The idea revolves around all the currenicies being dead and not much freedom of what is considered a shop.
> 
> (not main idea) new currency modal: The currency system is now refined and now actually uses it. If a shop joins a currency, the currency will require at least one item to be put in so currency is actually in circulation in there. The idea of inflation is also gone, as there is a max cap of overall currency and user's currency. Exclusives that use the currency cannot be given special treatment.
> 
> (main idea): A forum where people can freely make what they feel like is supposed to be in requests, and link everything together.

I got a better idea.

**Comment by Semisol** [comment]
An entire shop platform...

**Comment by AfterNetwork** [comment]
> An entire shop platform...
Yes. That's even better. We can name it Shourm. (mix of shop and forum).

**Comment by Semisol** [comment]
A website, you can do orders via a GUI, create currencies etc.

I would suggest a reputation system, you can use it to create stuff, as a currency thing.

You start up with 1 free shop.

You complete orders, you gain reputation.

You can delete your old shop to make a new one or make a new shop by removing some of your reputation.

You get a 25% refund on deletion or removal.

Mod removal removes some reputation, depending on the severity.

Currencies are automated, handling transactions. We will be offering an editor for special currency properties, maybe some kind of code-ish format. Currency is released to the economy in specific intervals until the cap is reached, and the creation of the currency defines the payment. It will reward some currency for actions done by staff. A currency can be cloned, and the settings are immutable post-creation unless a mod takes action.

**Comment by Semisol** [comment]
You release it, and its the final product.

**Comment by AfterNetwork** [comment]
Hopefully I get unbanned, I just posted a very long appeal.
And yes, that is awesome.

**Comment by Semisol** [comment]
A website, you can do orders via a GUI, create currencies etc.

I would suggest a reputation system, you can use it to create stuff, as a currency thing.

You start up with 1 free shop.

You complete orders, you gain reputation.

You can delete your old shop to make a new one or make a new shop by removing some of your reputation.

You get a 25% refund on deletion or removal.

Mod removal removes some reputation, depending on the severity.

Currencies are automated, handling transactions. We will be offering an editor for special currency properties, maybe some kind of code-ish format. Currency is released to the economy in specific intervals until the cap is reached, and the creation of the currency defines the payment. It will reward some currency for actions done by staff. A currency can be cloned, and the settings are immutable post-creation unless a mod takes action.

Shops can have premade templates or custom templates. We will hire content reviewers for the platform, handling paid payments, and order disputes.

Reviews will grant the owner and staff reputation, while high amounts of negative reviews will remove.

Reputation cannot be transferred, and account sharing is not allowed.

Currencies are allowed to impose transaction fees, removing coins out of economy over time.

**Comment by AfterNetwork** [comment]
Yes, scrtchverifier and email options as this is an **INDEPENDENT** platform.

**Comment by AfterNetwork** [comment]
What about the federations, the thing that claim to help grow your shop?

**Comment by AfterNetwork** [comment]
This is also a UNIVERSAL forum, meaning that you can post a shop and do all orders for free or prompt to use ACTUAL money where we get 10% of cuts to make some money to keep running the platform.

**Comment by AfterNetwork** [comment]
@Semisol

**Comment by Semisol** [comment]
Nope.

**Comment by Semisol** [comment]
Not actual money, or email. We can't post it on scratch then.

**Comment by Semisol** [comment]
But yeah, federations might have an order share and staff share option.

**Comment by Semisol** [comment]
@AfterNetwork
